two-factor.php

```diff
@@ -276,3 +276,17 @@ function wpcom_vip_two_factor_admin_notice() {
 	</div>
 	<?php
 }
+
+/**
+ * Jetpack Account Protection does not factor in a user's MFA status.
+ *
+ * If a user has MFA active, we should skip the Account Protection check.
+ */
+add_filter( 'jetpack_account_protection_user_requires_protection', 'wpcom_vip_two_factor_bypass_jetpack_account_protection', 10, 2 );
+function wpcom_vip_two_factor_bypass_jetpack_account_protection( $user_requires_protection, $user ) {
+	if ( Two_Factor_Core::is_user_using_two_factor( $user->ID ) ) {
```

Comment:
Do we need a class/method exists check here? Is `Two_Factor_Core` always available?

Reply:
In theory it should be available, but I'm not sure if there are edge cases when this could not be available (broken sites?)

Reply:
Implemented here: 8ab371d90c74b290162817540cfd307ad497a69f